doc/manual/manual.tex

```diff
@@ -45,6 +45,13 @@
   \end{center}
 }}
 
+
+% A few math commands to make typing partial derivatives at fixed conditions easier
```

Comment:
Sorry I didn't notice that earlier, but can you please remove these new commands and use the original commands below? I know it makes writing the equations a bit more cumbersome, but it makes reading them much easier, and other people will have it easier to modify things later on.